### ISSUE TITLE
Support passing in class names

### ISSUE THREAD

**Issue by cjke** [goopscoop/svg-to-react-cli#10]
A common pattern for components is passing in props, especially a className:

    <MyFancySvg className="Logo--sm" />

It would nice if the exported stateless function accepted props to be passed on to the parent svg node (much like width/height now). 

Happy again to PR this if you see this as a warranted change

**Comment by goopscoop** [comment]
That's a valid feature I think. Not sure how you'll want to handle the input for that in a bulk conversion. Do you have an implementation idea for that scenario?

**Comment by cjke** [comment]
The current output looks something like:

```
export default function MyFancySvg({width = '50px', height = '50px'}) {
  return (
    <svg viewBox="0 0 93.87 436.96" width={width} height={height}>
      <rect />
    </svg>
  );
}
```

I'm thinking the change would look something like:

```
export default function MyFancySvg(props) {
  // need something here to set the defaults for h/w back to 50px
  return (
    <svg viewBox="0 0 93.87 436.96" props={...props}>
      <rect />
    </svg>
  );
}
```

The class names wouldn't be resolved at conversation time, so I don't think it would impact multiple vs single (unless I am missing something!)

Using the second "props" example above, we could then do something like:

`<MyFancySvg height="50px" width="50px" className="Logo--sm" />`

**Comment by goopscoop** [comment]
That sounds good to me. Feel free to submit a PR on this. Thanks again :)

**Comment by cjke** [comment]
Closed by #12
